app/models/reservation.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Reservation < ApplicationRecord
-  belongs_to :screening, :user
-  has_many :tickets
-  validates_associated :tickets
+  belongs_to :screening
+  belongs_to :user, optional: true
```

Comment:
Does it mean you want to have a reservation without its owner? 🤔

Reply:
Just, for now. Yet I just wanted to have working reservation creation without user and assigning seats to reservation. As a next step I will take care of assigning user to reservation. I will edit description of this PR and mark that reservation for now works without assigning it to user :)